pandas/core/frame.py

```diff
@@ -1116,16 +1116,15 @@ def to_dict(self, orient='dict', into=dict):
         else:
             raise ValueError("orient '%s' not understood" % orient)
 
-    def to_gbq(self, destination_table, project_id, chunksize=10000,
-               verbose=True, reauth=False, if_exists='fail', private_key=None):
-        """Write a DataFrame to a Google BigQuery table.
-
-        The main method a user calls to export pandas DataFrame contents to
-        Google BigQuery table.
+    def to_gbq(
+            self, destination_table, project_id, chunksize=10000,
```

Comment:
Can you keep this on the previous line as it was? (to keep consistency, we don't use this style anywhere else in the file for `def` lines)